### PR TITLE
Superglue can be applied to chairs, beds, vehicles

### PR DESCRIPTION
**Pull request by DamianX**
Destroying the chair you're glued on leaves you without a butt.

:cl:
 * rscadd: Superglue can now be applied to chairs, beds and vehicles.

### CODE REVIEW

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Oh no
Clowns tactically supergluing their ass to clowncarts

**Comment by DrSnips** [comment]
It's not in OP but it should be noted that the ass removal also comes with a nice 40 brute damage to the groin which will likely cause internal bleeding as well as 10 burn damage. Since it is a limited use traitor item it's fine I'd say but it should be made clear. I think the wiz ass blast spell does the same.

File: code/game/objects/structures/stool_bed_chair_nest/bed.dm [hunk 1]
```diff
@@ -127,6 +137,26 @@
...
+	return ..()
+
+/obj/structure/bed/Destroy()
+	if(glued && is_locking(lock_type))
+		glued = FALSE // So that unlock_atom called in /atom/movable/Destroy can succeed
+		var/mob/living/carbon/human/locked = get_locked(lock_type)[1]
+		if(istype(locked) && locked.op_stage.butt != 4) // does the target have an ass
+			var/obj/item/clothing/head/butt/ass = new(locked.loc)
+			ass.transfer_buttdentity(locked)
+			locked.op_stage.butt = 4 //No having two butts.
```
Comment (ShiftyRail) [inline code/game/objects/structures/stool_bed_chair_nest/bed.dm]: I'm pretty sure I made a define about this one

Comment (unid15) [reply code/game/objects/structures/stool_bed_chair_nest/bed.dm]: the whole ass removal thing should be a proc in itself, too

**Pull request merged**